codalab/apps/web/static/js/search/ws_actions.js

```diff
@@ -98,13 +98,15 @@ var WorksheetActions =  function() {
                                 if (data.data.stdout){
                                     console.log(data.data.stdout);
                                     term.echo(data.data.stdout);
-                                    var references = data.data.structured_result['refs'];
-                                    Object.keys(references).forEach(function(k) {
-                                        $(".terminal-output div div:contains(" + k + ")").html(function(idx, html) {
-                                            var link = '/' + references[k]['type'] + 's/' + references[k]['uuid'];
-                                            return html.replace(k, "<a href=" + link + " target='_blank'>" + k + "</a>");
-                                        });
-                                    }, this);
+                                    if (data.data.structured_result && data.data.structured_result.refs){
+                                        var references = data.data.structured_result['refs'];
+                                        Object.keys(references).forEach(function(k) {
+                                            $(".terminal-output div div:contains(" + k + ")").html(function(idx, html) {
+                                                var link = '/' + references[k]['type'] + 's/' + references[k]['uuid'];
```

Comment:
If 'type' and 'uuid' don't exist, or 'type' is not 'bundle' or 'worksheet', we have a problem...